jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpan.java

```diff
@@ -61,7 +63,7 @@
     this.startTimeMicroseconds = startTimeMicroseconds;
     this.startTimeNanoTicks = startTimeNanoTicks;
     this.computeDurationViaNanoTicks = computeDurationViaNanoTicks;
-    this.tags = new HashMap<String, Object>();
+    this.tags = new ConcurrentHashMap<String, Object>();
```

Comment:
I think we could also remove the synchronized from the tags getter.

Reply:
I think it could be removed from other methods too. I will rather do it in a separate PR with more concurrent test and keep this clear. `synchronized` was not related to the concurrent modifications in this case.